modules/trusted-firmware-m/Kconfig

```diff
@@ -93,12 +120,17 @@ config TFM_CMAKE_BUILD_TYPE_DEBUG
 endchoice
 
 config TFM_ISOLATION_LEVEL
-	int "Isolation level setting."
+	int "Isolation level setting." if !(TFM_PROFILE_TYPE_SMALL || TFM_PROFILE_TYPE_MEDIUM || TFM_PROFILE_TYPE_LARGE)
```

Comment:
```suggestion
	int "Isolation level setting." if TFM_PROFILE_TYPE_NOT_SET
```

Reply:
Sure, I can do that. :)
I thought it was a bit more pedantic if I list all the existing profiles.
Will apply your fb now.

Reply:
done